docs/developer-handbook.md

```diff
@@ -19,6 +19,7 @@
   - [Body](#body)
   - [Footer](#footer)
   - [Examples](#examples)
+- [Coding Styles](#coding-styles)  
```

Comment:
Should be `Coding style` and `#coding-style` to match the heading so the anchor link works.

Reply:
Actually, run `tocdoc` to generate this.